suite-native/navigation/src/checkIsActiveRouteBlacklisted.ts

```diff
@@ -0,0 +1,19 @@
+import { navigationContainerRef } from './components/NavigationContainerWithAnalytics';
+import { DeviceOnboardingStackRoutes, RootStackRoutes } from './routes';
+
+type RouteType = RootStackRoutes | DeviceOnboardingStackRoutes;
+type Routes = RouteType[];
+
+const getActiveRouteName = () =>
+    navigationContainerRef.getState()?.routes.at(-1)?.name as RouteType;
+
+export const isActiveRouteAnyOf = ({ routeList }: { routeList: Routes }) => {
+    const activeRouteName = getActiveRouteName();
+
+    if (!activeRouteName) return false;
+
+    return routeList.includes(activeRouteName);
+};
+
+export const checkIsActiveRouteBlacklisted = ({ blacklist }: { blacklist: Routes }) =>
```

Comment:
Would now make sense to rename?
```suggestion
export const isActiveRouteBlacklisted = ({ blacklist }: { blacklist: Routes }) =>
```

Reply:
isActiveRouteBlacklisted doesn't evoke function name in my obinion, but variable name.

Reply:
But there was inconsistency in naming between those 2 functions. I prefixed both with `check..` now.

Reply:
Yeah, was referring to the inconsistency. 👍